include/EVT/manager.hpp

```diff
@@ -84,6 +87,14 @@ IWDG& getIWDG(uint32_t ms) {
     static IWDGf3xx iwdg(ms);
     return iwdg;
     #endif
+    #ifdef STM32F4xx
+    // TODO: Calculate range
```

Comment:
Calculate the time range that the class supports, according to the prescaler you picked. I'm pretty sure it will just match the one above